Dockerfile_mbtci_template

```diff
@@ -87,7 +100,6 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
       | sort -u \
       | xargs -r apt-mark manual \
     && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
-    && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
```

Comment:
cf. previous comment.

Reply:
I change nodejs install target path, from /usr/local/bin to /usr/local/nodejs. Same as go, from /usr/local/bin to /usr/local/go.

Making node and go installed different path is clear, and before we merge all docker file into one docker file Dockerfile_mbtci_template, the implementaion in Dockerfile_mbtci_java11, java8 and java 17, different modules are installed in different folder.

Install path is invisible to docker image users, why changing the install path may cause "backwards compatibility" ?

Reply:
The docker image must follow established standard expected by users to be followed like the FHS (https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard). 
You can create dedicated installation path for the archive installed but not in /usr/local. But it's unneeded and just add unneeded env vars, shell commands, etc. and must be done as part of another PR. 

For example `npm -g install` just follow the FHS and install binaries in /usr/local to keep things simple. Since all archives follow the FHS (with the expected permission) and are designed to be unarchived into /usr/local to avoid playing with PATH (and can lead to security issue), what is point of making things more complicated than it should at the cost of critical env var mangling such as PATH and insecure permissions change?

Reply:
Yes, I revert the code which install nodejs under path "/usr/local/nodejs", now nodejs is installed under "/usr/local"
https://github.com/SAP/cloud-mta-build-tool/pull/1023/commits/b3ea0b94387d71af5ac4e4952a133f0a78328619